pages/k8s/1.29/upgrading.md

```diff
@@ -15,7 +15,7 @@ toc: False
 
 It is recommended that you keep your **Kubernetes** deployment updated to the latest available stable version. You should also update the other applications which make up **Charmed Kubernetes**. Keeping up to date ensures you have the latest bug-fixes and security patches for smooth operation of your cluster.
 
-You can check the latest release version on the [Kubernetes release page on GitHub][k8s-release].
+New minor versions of **Kubernetes** are set to release three times per year. You can check the latest release version on the [Kubernetes release page on GitHub][k8s-release].
```

Comment:
I feel we ought to call out the 'integration gaps' stuff quite prominently here

Reply:
i put a big warning at the very top for both new + gap areas that will affect upgrades.